packages/orbit/src/normalizeAngle.js

```diff
@@ -2,10 +2,10 @@ import { TAU } from './commonCamera.js'
 
 const { PI } = Math
 
-/** Bring angle into range [0...2PI]
- * 
- * @param {Number} a - (Radians) angle that might need correcting
- * @returns 
+/**
+ * Bring angle into range [0...2PI]
```

Comment:
Seems that TAU is being used but the comment says PI :(

Reply:
@z3dev nice to have you here :) . Yes I was also curious about TAU and like the idea, so I looked to use it when I can ;) 

Since I am "test in production" type of guz, I fixed the comment to use TAU directly on main branch :)